serverless-compose.yml

```diff
@@ -7,21 +7,21 @@ services:
     path: src/services/mmdl
   seatool:
     path: src/services/seatool
-  status:
-    path: src/services/status
+  mmdl-status:
+    path: src/services/mmdl-status
   appian-status:
     path: src/services/appian-status
-  compare:
+  compare-mmdl:
```

Comment:
before this the changes I saw we're to `mmdl-compare`.  We should be consistent if the mmdl should go on the beginning or the end.

Reply:
This is a good point, I can change the appian namespaces over to whatever you land on @benjaminpaige

Reply:
yeah thats a good point jon. id say lets do [the thing is does] +  [the thing it does it for] so we lets do `status-mmdl` & `status-appian` cool? i can make that change here

Reply:
I'm good with that